sources-user.js

```diff
@@ -18,6 +18,15 @@ module.exports = [
       patterns: `docs/en/userguide/**`,
     }
   },
+  {
+    resolve: `gatsby-source-git`,
+    options: {
+      name: `user--5--optional_features/mfa`,
+      remote: `https://github.com/silverstripe/silverstripe-mfa`,
+      branch: `5.2`,
+      patterns: `docs/en/userguide/**`,
+    }
+  },
```

Comment:
```suggestion
```
MFA is already in this file, so this is a duplicate entry.

Reply:
Removed